website/docs/reference/dbt_project.yml.md

```diff
@@ -48,6 +53,10 @@ dbt uses YAML in a few different places. If you're new to YAML, it would be wort
 
 [require-dbt-version](/reference/project-configs/require-dbt-version): version-range | [version-range]
 
+[dbt-cloud](/docs/cloud/cloud-cli-installation):
```

Comment:
Can we just version this new info instead of duplicating the entire project file? This might make the code easier to read.